### PR TITLE
[Added] copy-to-clipboard TCET Linux website

### PR DESCRIPTION
**Pull request by MJ665**
Added content and image for copy-to-clipboard.mdx for TCET Linux website.

### CODE REVIEW

File: docs/projects/tcet-linux-website/frontend-components/copy-to-clipboard.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+title: Copy To Clipboard
+id: copy-to-clipboard
+description: Copy To Clipboard Component in TCET Linux Website
+sidebar_label: Copy To Clipboard
+keywords: [Copy to clipboard, Clipboard, useOnClickOutside, useRef, useEffect, Function component]
+---
+
+The Copy To Clipboard component is used in the [**fourth section**](https://linux.tcetmumbai.in/#verify) of the [**TCET Linux website**](https://linux.tcetmumbai.in/). The CopyToClipboard component provides a simple and convenient way to copy text to the clipboard. This is a common requirement in many applications, especially those that involve sharing or saving content. It provides visual feedback through the notification dialog, handles user interactions, and enhances the user experience by automatically hiding the dialog and closing it when clicked outside. By encapsulating the copy-to-clipboard functionality within a dedicated component, our codebase becomes more *organized* and *modular*. This improves code *maintainability*, *readability*, and *makes it easier for other developers to understand and modify* the code in the future.
```
Comment (himanshu-03) [inline docs/projects/tcet-linux-website/frontend-components/copy-to-clipboard.mdx:9]: bold out some important texts here and try to insert some hyperlinks for website or other things

**Review by himanshu-03 — CHANGES_REQUESTED** [review]
Make changes ASAP

File: docs/projects/tcet-linux-website/frontend-components/copy-to-clipboard.mdx (hunk 1 above)
Comment (MJ665) [reply docs/projects/tcet-linux-website/frontend-components/copy-to-clipboard.mdx]: done

**Review by himanshu-03 — APPROVED** [review]
LGTM!

**Pull request merged**